test/Microsoft.Framework.Internal.Test/PropertyHelperTest.cs

```diff
@@ -343,6 +343,102 @@ public void MakeFastPropertySetter_SetsPropertyValues_ForNewedProperties()
             Assert.Equal("NewedTest value", instance.PropB);
         }
 
+        [Fact]
+        public void MakeFastPropertyGetter_ReferenceType_ForNullObject_Throws()
+        {
+            // Arrange
+            var property = PropertyHelper
+                .GetProperties(typeof(BaseClass))
+                .Single(p => p.Name == nameof(BaseClass.PropA));
```

Comment:
nit: any reason for not using `MyProperties.StringProp` makes the code easier to read